apps/scan/frontend/src/app_root.tsx

```diff
@@ -184,33 +187,35 @@ export function AppRoot({
       </React.Fragment>
     );
     return (
-      <ScreenMainCenterChild>
-        <SystemAdministratorScreenContents
-          displayRemoveCardToLeavePrompt
-          logger={logger}
-          primaryText={
-            <React.Fragment>
-              To adjust settings for the current election,
-              <br />
-              please insert an Election Manager or Poll Worker card.
-            </React.Fragment>
-          }
-          unconfigureMachine={() => unconfigureMutation.mutateAsync()}
-          resetPollsToPausedText="The polls are closed and voting is complete. After resetting the polls to paused, it will be possible to re-open the polls and resume voting. All current cast vote records will be preserved."
-          resetPollsToPaused={
-            pollsState === 'polls_closed_final'
-              ? () =>
-                  transitionPollsMutation.mutateAsync({
-                    type: 'pause_voting',
-                    time: Date.now(),
-                  })
-              : undefined
-          }
-          isMachineConfigured={Boolean(electionDefinition)}
-          usbDriveStatus={usbDrive}
-          additionalButtons={additionalButtons}
-        />
-      </ScreenMainCenterChild>
+      <Screen title="System Administrator">
```

Comment:
Should this likewise be moved into its own file?

Reply:
Sure, can do